api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -279,6 +279,9 @@ message AccessListMemberMetadata {
 
   // Members are all members affected by the access list membership change.
   repeated AccessListMember Members = 2 [(gogoproto.jsontag) = "members,omitempty"];
+
+  // AccessListTitle is the access list's title.
+  string AccessListTitle = 4 [(gogoproto.jsontag) = "access_list_title,omitempty"];
```

Comment:
Is _title_ just another word for _name_ in this case?

Reply:
Access List spec has a separate `title` field for "display name". The name `metadata.name` gets uuid.